test/functional/inspec_init_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'functional/helper'
+require 'fileutils'
+require 'tmpdir'
+
+describe 'inspec init' do
+  include FunctionalHelper
+
+  tmpdir = Dir.tmpdir
+  puts tmpdir
+
+  describe 'inspec init profile with/slash' do
+    it 'prevents profile names without valid characters' do
```

Comment:
This test isn't clear to me.  I thought the intent was to allow slashes in the arg, but only use the last path component.  The test name says we are preventing it, but the this seems to explicitly allow it, checking for exit code 0.